src/main/scala/org/broadinstitute/dsde/firecloud/filematch/strategy/OntSingleReadStrategy.scala

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.dsde.firecloud.filematch.strategy
+
+import org.broadinstitute.dsde.firecloud.filematch.result.{FailedMatchResult, FileMatchResult, SuccessfulMatchResult}
+import OntSingleReadStrategy.PATTERN
+
+import java.nio.file.{Path, Paths}
+import scala.util.matching.Regex
+
+class OntSingleReadStrategy extends FileRecognitionStrategy {
+
+  override def matchFirstFile(path: Path): FileMatchResult =
+    PATTERN.findFirstMatchIn(path.toString) match {
+      case Some(regexMatch) => SuccessfulMatchResult(path, Paths.get(""), regexMatch.group(1))
```

Comment:
Should this be a `PartialMatchResult` instead of passing in an empty path for the second file?

Reply:
Using a `PartialMatchResult` causes the matcher to generate an overall `FailedMatchResult`. To address this, I overloaded `SuccessfulMatchResult` to optionally accept two parameters (filePath1, id), as shown here: https://github.com/broadinstitute/firecloud-orchestration/pull/1581/commits/6713d87751ef556da69ebbbd4b7c461f178cebd4